### PR TITLE
init package

### PR DESCRIPTION
**Pull request by AnyaCoder**
**Is this PR adding new feature or fix a BUG?**

Add feature.

**Is this pull request related to any issue? If yes, please link the issue.**

Initial Version of Package.

### CODE REVIEW

File: fish_speech/datasets/text.py
Comment (leng-yue) [inline fish_speech/datasets/text.py:1]: Why we need to replace mask our with text-labeler?

File: fish_speech/train.py
Comment (leng-yue) [inline fish_speech/train.py:1]: This should be override in launching script: `trainer.strategy.process_group_backend=gloo`

File: tools/webui.py
Comment (leng-yue) [inline tools/webui.py:1]: should try import spaces, this is important for huggingface spaces.

```python
try:
    import spaces
    GPU_DECORATOR = spaces.GPU
except ImportError:
    def GPU_DECORATOR(func):
        def wrapper(*args, **kwargs):
            return func(*args, **kwargs)
        return wrapper
```

**Review by leng-yue — CHANGES_REQUESTED** [review]
Nice job overall!

File: fish_speech/datasets/text.py
Comment (AnyaCoder) [reply fish_speech/datasets/text.py:1]: Ahh, thanks to pycharm's renaming

Comment (AnyaCoder) [reply fish_speech/datasets/text.py:1]: Ahh, thanks to pycharm's renaming

File: tools/webui.py
Comment (AnyaCoder) [reply tools/webui.py:1]: ok

File: fish_speech/train.py
Comment (AnyaCoder) [reply fish_speech/train.py:1]: image
image
Seems not okay...

Comment (AnyaCoder) [reply fish_speech/train.py:1]: OK，I‘ve resolved the problem by adding some params in `yaml`

**Review by leng-yue — APPROVED** [review]

**Pull request merged**